packit_service/worker/jobs.py

```diff
@@ -49,6 +49,16 @@ def add_to_mapping(kls: Type["JobHandler"]):
     return kls
 
 
+def do_we_process_fedmsg_topic(topic: str) -> bool:
+    """ do we process selected fedmsg topic? """
+    global JOB_NAME_HANDLER_MAPPING
```

Comment:
[nitpicking]: do we need this when we don't change the value? (we just iterate over it)

Reply:
I don't know.